application/src/main/java/org/togetherjava/tjbot/commands/moderation/ModerationUtils.java

```diff
@@ -372,4 +372,37 @@ static Optional<TemporaryData> computeTemporaryData(String durationText) {
      */
     record TemporaryData(Instant expiresAt, String duration) {
     }
+
+    /**
+     * Gives out advice depending on the {@link ModerationAction} and the parameters passed into it.
+     *
+     * @param action gives out a {@link ModerationAction} to be able to respond and build a message
+     *        appropriately such as {@link ModerationAction#MUTE} etc.
+     * @param message is the message that is being used to build responses, such as
+     *        {@link Guild#getName()}, and reasoning for an action.
```

Comment:
this feels fragile and confusing.
lets first understand everything that it can contain:
case 1 (length == 3)
* [0] -> duration message
* [1] -> server
* [2] -> reason

case (unmute, unquarantine):
* [0] -> server
* [1] -> reason

else (which actions are left over?)
* [0] -> server
* [1] -> reason

so. what i see: all of them must use server and reason. so u should take them as individual arguments. and then, some of the actions have a duration message (the temporary actions). this should be a nullable argument.

or, even better. the method should construct the duration message itself, based on which actions are temporary and which arent.

Reply:
Done